apps/api/src/app/shared/services/in-memory-provider/in-memory-provider.service.spec.ts

```diff
@@ -100,14 +108,14 @@ describe('In-memory Provider Service', () => {
 
         const { inMemoryProviderClient } = inMemoryProviderService;
 
-        expect(inMemoryProviderClient!.status).to.eql('connecting');
+        expect(inMemoryProviderClient!.status).to.eql('ready');
         expect(inMemoryProviderClient!.isCluster).to.eql(true);
 
         const options = inMemoryProviderService.getOptions();
         expect(options).to.eql(undefined);
 
         const clusterOptions = inMemoryProviderService.getClusterOptions();
-        expect(clusterOptions!.enableOfflineQueue).to.eql(true);
+        expect(clusterOptions!.enableOfflineQueue).to.eql(false);
```

Comment:
Update check due to change of configuration to the Redis Cluster provider to align it with the Elasticache Cluster provider configuration deployed in our environments.